code/game/machinery/doors/firedoor.dm

```diff
@@ -119,7 +350,7 @@
 			C.play_tool_sound(src)
 			boltslocked = !boltslocked
 			return
-	return ..()
+	. = ..()
```

Comment:
?

Reply:
isnt . = ..() preferred over return ..() ?

Reply:
Not at the end, no.